test/README.md

```diff
@@ -6,19 +6,13 @@ This folder contains nightly regression tests execututed against TorchServe mast
 
 ![Build Status](https://codebuild.us-east-1.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiS1QvY3lIUEdUb3hZVWNnbmJ2SEZCdExRNmNkNW9EVk1ZaFNldEk4Q0h3TU1qemwzQ29GNW0xMGFhZkxpOFpiMjUrZVVRVDUrSkh2ZDhBeFprdW5iNjRRPSIsIml2UGFyYW1ldGVyU3BlYyI6IjlvcjRqSTNMTmNhcExZbUwiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=staging_0_1_1)
 
-[Test Logs](https://torchserve-regression-test.s3.amazonaws.com/torch-serve-regression-test/tmp/test_exec.log)
+[Latest Test Logs](https://torchserve-regression-test.s3.amazonaws.com/public/latest/test_exec.log)
 
-### Running the test manually.
-
-Clone Torch Serve Repo & Build the Docker Image for the execition env.
+[Test Logs By Date](https://torchserve-regression-test.s3.amazonaws.com/public/TS-NIGHTLY-REGRESSION-2020-05-20/tmp/test_exec.log) - Update Date in URL
 
-```
-git clone https://github.com/pytorch/serve
-cd serve
-./build_image.sh
-```
+### Running the test manually.
 
-This would build a docker Image with a pytorch/torchserve:latest in which we would run our Regression Tests.
+Use Public TorchServe Docker Image for the execition env.
```

Comment:
Use of public docker image possess risk that if your current branch has introduced new dependencies then those won't available in that image.

Reply:
Changed this to use build the docker image locally.